lib/fathead/pip/fetch.py

```diff
@@ -0,0 +1,33 @@
+__author__ = 'kchahal'
+#To change this template use Tools | Templates.
+
+baseUrl = 'https://pip.pypa.io/en/stable/reference/'
+import urllib2
```

Comment:
After switching this to `urllib3`, I'm seeing errors. I'm testing with Python 3.5.2:

```
Traceback (most recent call last):
  File "fetch.py", line 4, in <module>
    response = urllib3.urlopen(baseUrl)
AttributeError: module 'urllib3' has no attribute 'urlopen'
```